Test/build.ps1.Tests.ps1

```diff
@@ -2,10 +2,14 @@
 #Requires -Version 5.1
 Set-StrictMode -Version 'Latest'
 
+$testRoot = $null
 $buildPs1Path = Join-Path -Path $PSScriptRoot -ChildPath '..\Whiskey\build.ps1' -Resolve
+
 function Init
 {
     Get-Module 'PowerShellGet','Whiskey','PackageManagement' | Remove-Module -Force -ErrorAction Ignore
+    $script:testRoot = Join-Path -Path $TestDrive.FullName -ChildPath ([IO.Path]::GetRandomFileName())
+    New-Item -Path $testRoot -ItemType 'Directory' | Out-Null

```

Comment:
This is on purpose. These tests don't use WhiskeyTest or any other module on purpose so that it can properly test that build.ps1 bootstraps itself. I didn't want a dependency to accidentally get installed/used and build.ps1 pass its tests but not actually work.